src/runtime/converter.cs

```diff
@@ -344,7 +344,7 @@ internal static IntPtr ToPythonImplicit(object value)
                 }
                 if (mt is ClassBase)
                 {
-                    result = ((ClassBase)mt).type;
+                    result = ((ClassBase)mt).type.Value;
```

Comment:
probably should RaiseTypeError if invalid here

Reply:
I think it's not possible to have an invalid value here as the above call to GetManagedObject would return null. Still added it as a paranoia check